examples/05_stable_diffusion/src/pipeline_stable_diffusion_ait_alt.py

```diff
@@ -13,532 +13,43 @@
 #  limitations under the License.
 #
 import inspect
-
 import os
 import re
-from typing import List, Optional, Union
-
+import PIL
 import torch
-from aitemplate.compiler import Model
 
-from diffusers import AutoencoderKL, EulerDiscreteScheduler, UNet2DConditionModel
+import numpy as np
 
+from typing import List, Optional, Union
+from aitemplate.compiler import Model
 from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
 from diffusers.utils.pil_utils import numpy_to_pil
 from tqdm import tqdm
-
 from transformers import CLIPTextConfig, CLIPTextModel, CLIPTokenizer
 
 from .compile_lib.compile_vae_alt import map_vae_params
 from .modeling.vae import AutoencoderKL as ait_AutoencoderKL
+from .pipeline_utils import convert_ldm_unet_checkpoint, convert_ldm_vae_checkpoint
 
+from diffusers import (
```

Comment:
Please fix lint errors.

Reply:
What's wrong? Can't quite figure out

Reply:
Check the log of [lint action](https://github.com/facebookincubator/AITemplate/actions/runs/5247622075/jobs/9627067379?pr=762)
```
Error: ./examples/05_stable_diffusion/src/pipeline_stable_diffusion_ait_alt.py:34:1: F401 'diffusers.DDIMScheduler' imported but unused
Error: ./examples/05_stable_diffusion/src/pipeline_stable_diffusion_ait_alt.py:34:1: F401 'diffusers.PNDMScheduler' imported but unused
Error: ./examples/05_stable_diffusion/src/pipeline_stable_diffusion_ait_alt.py:34:1: F401 'diffusers.EulerDiscreteScheduler' imported but unused
Error: ./examples/05_stable_diffusion/src/pipeline_stable_diffusion_ait_alt.py:789:13: F841 local variable 'offset' is assigned to but never used
```

Reply:
oh thanks, gonna fix